app/javascript/stylesheets/_email.scss

```diff
@@ -627,7 +626,10 @@ $accent-color: map-get($colors, "pink"); // [1]
 
 .price {
   display: inline-block;
-  background: $accent-color url("~images/email/price_tag_arrow.png") no-repeat right / auto 100%; // [1] [4]
+  border: $border;
+  border-right: none;
+  padding: spacer(1) spacer(5) spacer(1) spacer(2);
+  background: $accent-color url("~images/email/price_tag_arrow.png") no-repeat right / contain;
```

Comment:
Changed from `auto 100%` to `contain` to fix the single pixel bar of pink you can see in the before image for the email.